rules/sqlperf-select-required-columns/rule.md

```diff
@@ -0,0 +1,35 @@
+---
+seoDescription: Do you only SELECT the columns you require?
+type: rule
+archivedreason:
+title: Do you only SELECT the columns you require?
+guid: db46290f-e1ab-4279-8833-f5a1d54c529f
+uri: sqlperf-select-required-columns
+created: 2024-07-23T07:26:54.0000000Z
+authors:
+  - title: Bryden Oliver
+    url: https://ssw.com.au/people/bryden-oliver
+related:
+  - sqlperf-reduce-table-size
+  - sqlperf-verify-indexes-used
+  - sqlperf-avoid-implicit-type-conversions
+  - sqlperf-avoid-looping
+  - sqlperf-use-and-instead-of-or
+  - sqlperf-minimise-large-writes
+---
+
+Do you only SELECT the columns that your application is about to use?
```

Comment:
@brydeno 

Can you replace the repeated tiltle with a sentence that summarizes the goal of the rule?

Reply:
Done.